src/hotspot/cpu/riscv/vm_version_riscv.cpp

```diff
@@ -106,6 +106,12 @@ void VM_Version::initialize() {
     FLAG_SET_DEFAULT(UseMD5Intrinsics, false);
   }
 
+  // compressed instruction extension
```

Comment:
I think this should better to place this one after UseRVV and UseRVB.

Reply:
done. Thanks for pointing out this. It is inherited from my original patch and it indeed needs to be moved downward to keep the consistency.